### PR TITLE
chore(java): rearrange code

### PR DESCRIPTION
**Pull request by menski**
## Description

Let intellij code formatter rearrange code

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]
Seems like after rearranging some things were misformatted - should look fine now. Also it was producing a compile error in `GatewayTest` due to badly rearranging fields for some reason

File: transport/src/main/java/io/zeebe/transport/impl/ServerResponseImpl.java [hunk 1]
```diff
@@ -72,4 +58,18 @@ public BufferWriter getWriter() {
   public long getRequestId() {
```
Comment (Zelldon) [inline transport/src/main/java/io/zeebe/transport/impl/ServerResponseImpl.java]: Why is this for example splitted with the setter :sweat_smile:

File: logstreams/src/test/java/io/zeebe/logstreams/util/SyncLogStream.java [hunk 2]
```diff
@@ -73,4 +63,14 @@ public void setCommitPosition(final long position) {
   public LogStreamReader newLogStreamReader() {
     return logStream.newLogStreamReader().join();
   }
+
+  @Override
+  public LogStreamRecordWriter newLogStreamRecordWriter() {
```
Comment (Zelldon) [inline logstreams/src/test/java/io/zeebe/logstreams/util/SyncLogStream.java]: Which setting exactly we used now? It seems for me it is lexicographical method ordering ?

**Pull request merged**